src/components/dashboard/AddTaskModal.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from "react";
```

Comment:
Why is component required here?

Reply:
while createing class component React.component is used